docs/data.rst

```diff
@@ -1,8 +1,11 @@
-Input Specifications **(IN PROGRESS)**
-======================================
+Data Specifications
```

Comment:
I would leave this as `input`, since we have an `output specifications` section later.